lib/Const.ts

```diff
@@ -135,7 +141,7 @@ export const getMicrosoftAuthorizeUrl = (aadTenantId: string) => {
 };
 
 export const getGraphApiRevokeRefreshTokenUrl = () => {
-    return `${GraphApiBaseUrl}/${GraphApiVersion.V1}/${GraphApiEndpoint.RevokeRefreshToken}`;
+    return `${GraphApiBaseUrl}/${GraphApiVersion.V1}/${GraphApiEndpoint.RevokeRefreshToken}?$format=json`;
```

Comment:
I'm definitely not sure about this one, but just wanted to double check that the `$` really belongs there and isn't a typo :stuck_out_tongue: I haven't seen many query params with this character, and thought  we may need to `encodeURIComponent` that one